internal/protocol/version.go

```diff
@@ -7,13 +7,15 @@ import (
 // VersionNumber is a version number as int
 type VersionNumber int
 
-// gquicVersion0 is the "base" for gQUIC versions
-// e.g. version 39 is gquicVersion + 0x39
-const gquicVersion0 = 0x51303300
+// gQUIC version range as defined in the wiki: https://github.com/quicwg/base-drafts/wiki/QUIC-Versions
+const (
+	gquicVersion0   = 0x51303030
+	maxGquicVersion = 0x51303439
```

Comment:
That seems quite low – why not just 0x5130ffff?

Reply:
Just following the WG wiki here: https://github.com/quicwg/base-drafts/wiki/QUIC-Versions. We can extrapolate, if you want to.